Dockerfile.buster

```diff
@@ -54,8 +54,12 @@ RUN \
   && echo "deb-src https://deb.nodesource.com/${NODE_VERSION} buster main" | tee -a /etc/apt/sources.list.d/nodesource.list \
   && apt-get update \
   && apt-get install -y --no-install-recommends nodejs \
+  && curl -sS https://downloads.mariadb.com/MariaDB/mariadb_repo_setup | bash \
```

Comment:
Can you add a comment with a link to the MariaDB instructions to install like this?

I'd prefer to not pipe a script into bash, but from looking at their docs they don't offer any alternative instructions, we'd have to pull the steps out of that script if we wanted to do that

Reply:
IIRC it is from their docs but ➕  to adding a link to them for future reference.

Reply:
https://mariadb.com/kb/en/mariadb-package-repository-setup-and-usage/ but per Kyle's suggestion let's try removing.